adafruit_bitmapsaver.py

```diff
@@ -38,18 +39,18 @@
 
 def _write_bmp_header(output_file, filesize):
     output_file.write(bytes("BM", "ascii"))
-    output_file.write(struct.pack("<I", filesize))
+    output_file.write(struct.pack("<I", filesize))  # pylint: disable=no-member
     output_file.write(b"\00\x00")
     output_file.write(b"\00\x00")
-    output_file.write(struct.pack("<I", 54))
+    output_file.write(struct.pack("<I", 54))  # pylint: disable=no-member
 
 
 def _write_dib_header(output_file, width, height):
-    output_file.write(struct.pack("<I", 40))
-    output_file.write(struct.pack("<I", width))
-    output_file.write(struct.pack("<I", height))
-    output_file.write(struct.pack("<H", 1))
-    output_file.write(struct.pack("<H", 24))
+    output_file.write(struct.pack("<I", 40))  # pylint: disable=no-member
+    output_file.write(struct.pack("<I", width))  # pylint: disable=no-member
+    output_file.write(struct.pack("<I", height))  # pylint: disable=no-member
+    output_file.write(struct.pack("<H", 1))  # pylint: disable=no-member
+    output_file.write(struct.pack("<H", 24))  # pylint: disable=no-member
```

Comment:
Why were these pylint disables added? Was it failing the pylint for you without these?

Reply:
it fails pre-commit hooks
```
pylint (library code)....................................................Failed
- hook id: pylint
- exit code: 2

************* Module adafruit_bitmapsaver
adafruit_bitmapsaver.py:55:0: E0011: Unrecognized file option 'ignore' (unrecognized-inline-option)
adafruit_bitmapsaver.py:48:22: E1101: Module 'struct' has no 'pack' member (no-member)
adafruit_bitmapsaver.py:51:22: E1101: Module 'struct' has no 'pack' member (no-member)
adafruit_bitmapsaver.py:55:22: E1101: Module 'struct' has no 'pack' member (no-member)
adafruit_bitmapsaver.py:56:22: E1101: Module 'struct' has no 'pack' member (no-member)
adafruit_bitmapsaver.py:57:22: E1101: Module 'struct' has no 'pack' member (no-member)
adafruit_bitmapsaver.py:58:22: E1101: Module 'struct' has no 'pack' member (no-member)
adafruit_bitmapsaver.py:59:22: E1101: Module 'struct' has no 'pack' member (no-member)